metrics.go

```diff
@@ -8,6 +8,7 @@ import (
 
 var (
 	statusCodes                    *prometheus.CounterVec
+	statusCodesProxy               *prometheus.CounterVec
```

Comment:
a bit picky but you should rename this variable statusCodesClickhouse since statusCodesProxy could be seen as "the statusCodes from chproxy)

Reply:
renamed the metric https://github.com/ContentSquare/chproxy/pull/407/commits/c3ebc353912d8c4f5544a6e25fcf09b24ceddd3c